python/pyarrow/tests/parquet/test_encryption.py

```diff
@@ -236,6 +236,35 @@ def kms_factory(kms_connection_configuration):
         # Write with encryption properties
         write_encrypted_parquet(path, data_table, encryption_config,
                                 kms_connection_config, crypto_factory)
+        
+def test_encrypted_parquet_write_col_key_and_uniform_encryption(tempdir, data_table):
+    """Write an encrypted parquet, but give both column key and uniform encryption.
+    
+    This should raise an error.
+    """
+    path = tempdir / 'encrypted_table_no_col_key.in_mem.parquet'
```

Comment:
If we're specifying a file name we should be careful not to reuse the same name as in the previous test.